app/models/authority.rb

```diff
@@ -220,7 +220,8 @@ def morph_url
 
   def morph_feed_url_for_date_range(start_date, end_date)
     query = CGI.escape("select * from `data` where `date_scraped` >= '#{start_date}' and `date_scraped` <= '#{end_date}'")
-    "https://api.morph.io/#{morph_name}/data.json?query=#{query}&key=#{ENV['MORPH_API_KEY']}"
+    key = CGI.escape ENV['MORPH_API_KEY'] || ""
```

Comment:
> Since morph.io requires an API key it seems reasonable to throw an exception if it's not set.

I really like this idea, it provides an opportunity to give some helpful advice for a developer in this situation to point the in the right direction.